src/actors/runtime/AbstractActor.ts

```diff
@@ -71,7 +71,7 @@ export default abstract class AbstractActor {
    * @param <Type> Type of the state object
    * @return Async void response
    */
-  async registerActorReminder<Type>(reminderName: string, dueTime: Temporal.Duration, period: Temporal.Duration, state?: any) {
+  async registerActorReminder<_Type>(reminderName: string, dueTime: Temporal.Duration, period: Temporal.Duration, state?: any) {
```

Comment:
Why are we using underscores?

Reply:
_XYZ is a general convention for args/params that are unused. I have added the same to eslintrc file too.

Reply:
Could you go a bit more in depth on which linting rule this solves? It's a bit new for me the underscore usage. As far as I could find it is related to Python a lot, and when I research it for Typescript, they often have a "Ban leading underscore" rule. So I am unsure this is the way to go personally

Reply:
The following in `.eslintrc.json` ignores that.
```json
        "@typescript-eslint/no-unused-vars": ["warn", {
            "varsIgnorePattern": "^_",
            "argsIgnorePattern": "^_"
        }]
```
As far as where the standard comes from comes, I saw it at multiple places. Example, https://github.com/microsoft/TypeScript/issues/9458  https://stackoverflow.com/questions/38835001/skip-type-check-on-unused-parameters

Reply:
Ok, thanks for clarifying!